src/Nethermind/Nethermind.Core/Extensions/UInt64Extensions.cs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+
+namespace Nethermind.Core.Extensions;
+
+public static class UInt64Extensions
```

Comment:
Looks unused apart from tests
Also `BitConverter.ToUInt64` or `Unsafe.As` is probably better?